suites/ceph-ansible/smoke/2-setup/ceph_ansible.yaml

```diff
@@ -0,0 +1,3 @@
+tasks:
```

Comment:
Not all suites repeat the same base set of tasks:
https://github.com/ceph/ceph-qa-suite/blob/master/suites/rbd/basic/base/install.yaml